contracts/solidity/test/Transfer.sol

```diff
@@ -10,7 +10,7 @@ contract TransferTest is Test {
     function setUp() public {
         address snMessagingAddress = 0xde29d060D45901Fb19ED6C6e959EB22d8626708e;
         uint256 snEscrowAddress = 0x0;
-        uint256 snEscrowWithdrawSelector = 0x0;
+        uint256 snEscrowWithdrawSelector = 0x15511cc3694f64379908437d6d64458dc76d02482052bfb8a5b33a72c054c77;
```

Comment:
Should we consider adding this to the constructor. What happens if we change the signature of the `withdraw_with_fallback` function in the starknet contract?

Reply:
we concluded the best way to apply this fix is in PR #46 , since this functionality is migrated somewhere else in that PR